data/tika.xml

```diff
@@ -7060,7 +7060,7 @@
   </mime-type>
 
   <mime-type type="text/x-idl">
-    <_comment>Inteface Definition Language</_comment>
+    <_comment>Interface Definition Language</_comment>
```

Comment:
These come from an upstream Tika file, so we should leave them as-is or push changes upstream.